src/resource/resource.test.ts

```diff
@@ -52,6 +43,16 @@ import {
 } from "../interfaces";
 import { createSolidDataset } from "./solidDataset";
 
+jest.mock("../fetcher.ts", () => ({
+  fetch: jest.fn().mockImplementation(() =>
+    Promise.resolve(
+      new Response(undefined, {
+        headers: { Location: "https://arbitrary.pod/resource" },
+      })
+    )
+  ),
+}));
+
```

Comment:
Depending on how soon fetch mocking is updated it might be worth factoring this into a factory function or running it in the global jest environment setup

Reply:
We still need to plan it, and it's quite a big undertaking. But I do wanna see it resolved. (just not in this PR)